api_app/README.md

```diff
@@ -267,3 +267,15 @@ Some workspace routes require `authConfig` field in the request body. The AAD sp
 ```
 
 > **Note:** The app registration for a workspace is not created by the API. One needs to be present (created manually) before using the API to provision a new workspace.
+
+## Network requirements
+
+To be able to run the TRE API it needs to acccess the following resource outside the Azure TRE VNET via explicit allowed [Service Tags](https://docs.microsoft.com/en-us/azure/virtual-network/service-tags-overview) or URLs.
+
+| Service Tag / Destination | Justification |
+| --- | --- |
+| AzureActiveDirectory | Authenticate with the User Assigned identity to access Azure Cosmos DB and Azure Service Bus. |
+| AzureMonitor | Publish traces and logs to one central place for troubleshooting. |
```

Comment:
Have we got an issue to look at moving this to a private endpoint?

Reply:
We do #742